.travis.yml

```diff
@@ -1,15 +1,33 @@
-language: haskell
+# https://docs.haskellstack.org/en/stable/travis_ci/
```

Comment:
Oh, OK. Certainly, being able to build the project is pretty important! Sorry to hear you got yourself into that unfortunate non-linux situation.